service/src/main/java/greencity/mapping/HabitTranslationMapper.java

```diff
@@ -18,6 +18,37 @@ protected HabitTranslation convert(HabitTranslationDto habitTranslationDto) {
             .build();
     }
 
+    /**
+     * Additional method that build {@link HabitTranslation} from
+     * {@link HabitTranslationDto} but from nameUa, descriptionUa, habitItemUa
+     * fields if they not null.
+     *
+     * @param habitTranslationDto {@link HabitTranslationDto}
+     * @return {@link HabitTranslation}
+     *
+     * @author Chernenko Vitaliy
+     */
+    public HabitTranslation convertUa(HabitTranslationDto habitTranslationDto) {
+        HabitTranslation habitTranslation = new HabitTranslation();
+        if (habitTranslationDto.getNameUa() == null) {
```

Comment:
that logic can be simplified using  [defaultIfNull](https://commons.apache.org/proper/commons-lang/javadocs/api-3.1/src-html/org/apache/commons/lang3/ObjectUtils.html#line.93)

Reply:
Very useful thing. I think I'll use it in the future.
Thanks. Fixed.